sdk/tests/wallet/transactions.rs

```diff
@@ -1,42 +1,43 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-// use iota_sdk::wallet::{MintNftParams, Result, SendNftParams, SendParams, TransactionOptions};
-// use pretty_assertions::assert_eq;
+use iota_sdk::wallet::SendParams;
+use pretty_assertions::assert_eq;
 
-// use crate::wallet::common::{make_wallet, request_funds, setup, tear_down};
+use crate::wallet::common::{make_wallet, request_funds, setup, tear_down};
 
-// #[ignore]
-// #[tokio::test]
-// async fn send_amount() -> Result<(), WalletError> {
-//     let storage_path_0 = "test-storage/send_amount_0";
-//     setup(storage_path_0)?;
-//     let storage_path_1 = "test-storage/send_amount_1";
-//     setup(storage_path_1)?;
+#[ignore]
+#[tokio::test]
+async fn send_amount() -> Result<(), Box<dyn std::error::Error>> {
+    let storage_path_0 = "test-storage/send_amount_0";
+    setup(storage_path_0)?;
+    let storage_path_1 = "test-storage/send_amount_1";
+    setup(storage_path_1)?;
 
-//     let wallet_0 = make_wallet(storage_path_0, None, None).await?;
-//     request_funds(&wallet_0, 1).await?;
+    let wallet_0 = make_wallet(storage_path_0, None, None).await?;
+    request_funds(&wallet_0).await?;
 
-//     let wallet_1 = make_wallet(storage_path_1, None, None).await?;
+    let wallet_1 = make_wallet(storage_path_1, None, None).await?;
 
-//     let amount = 1_000_000;
-//     let tx = wallet_0
-//         .send_with_params([SendParams::new(amount, wallet_1.address().clone())?], None)
-//         .await?;
+    let amount = 1_000_000;
+    let tx = wallet_0
+        .send_with_params([SendParams::new(amount, wallet_1.address().await)?], None)
+        .await?;
 
-//     wallet_0
-//         .wait_for_transaction_acceptance(&tx.transaction_id, None, None)
-//         .await?;
+    wallet_0
+        .wait_for_transaction_acceptance(&tx.transaction_id, None, None)
+        .await?;
 
-//     let balance = wallet_1.sync(None).await.unwrap();
-//     assert_eq!(balance.base_coin().available(), amount);
+    let balance = wallet_1.sync(None).await.unwrap();
+    assert_eq!(balance.base_coin().available(), amount);
 
-//     tear_down(storage_path)
-// }
+    tear_down(storage_path_0)?;
+    tear_down(storage_path_1)
+}
 
 // #[ignore]
 // #[tokio::test]
-// async fn send_amount_127_outputs() -> Result<(), WalletError> {
+// async fn send_amount_127_outputs() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Why are the remaining not uncommented?

Reply:
Just didn't want to have so many changes at once

Reply:
Ok